Workload-Security/Integration/aws-python-amazon-inspector-integration/README.md

```diff
@@ -0,0 +1,24 @@
+Amazon Inspector integration with Cloud One Workload Security
+
+Background Information
+
+Amazon Inspector is the assessment service which is for security issues and vulnerabilities to help improve the security and compliance in AWS environment. Inspector continuously scans your AWS workloads for vulnerabilities. AWS Inspector maintains a knowledge base of rules including two types of assessment, Network Based Type and Host Based Type. We are more interested in the Host Based Type which includes Common vulnerability exposures (CVEs).
+Trend Micro Cloud One Workload Security IPS rules protect against Common vulnerability exposures (CVEs). Each rule has a designated CVEs that it protects against.
+In this case Amazon Inspector is acting as IDS (Intrusion Detection) and Trend Micro Cloud One Workload Security IPS is acting as IPS (Intrusion Protection).
+Purpose and Objective
+
+The goal of this integration is to be able to show Trend Micro customers that are using Amazon Inspector what the Amazon findings (CVEs) are being protected by IPS rules. In order to achieve this goal, we may need to let customers understand the security risk / vulnerability issue in their environment, then we could have some opportunity to adopt a C1WS solution to mitigate / solve their issue using the ips rules.
+This deployment will help customer in need or that are using Inspector to better link both cloud one Workload Security and Amazon Inspector.
+The functionality of the scripts
+
+In this project we have two deployments. The first deployment (vulnerability report) is responsible to send out a report of the vulnerabilities that are being protected by ips rules among the findings (CVEs) Inspector is detecting. The second deployment (auto assign ips rule) is responsible for assigning the ips rules found based on the Inspector findings to the computers (EC2 instances). This means if the Inspector detected any findings (CVEs) then the script will send information to Trend Micro Workload Security who will protect the instances from whatever was detected?
+Deployment
+

```

Comment:
I would add more details to the readme page. Ensure that all prereqs are listed before hand. Such as SES set up. Cloud One Api Key generation.